deploy-web/src/utils/deploymentData/index.ts

```diff
@@ -1,22 +1,5 @@
 import * as v1beta3 from "./v1beta3";
-import { mainnetId, testnetId, sandboxId } from "../constants";
 export * from "./helpers";
 
-export let deploymentData;
+export const deploymentData = v1beta3;
 export let selectedNetworkId: string;
-
-export function initDeploymentData() {
```

Comment:
you mean, you manually update the code?

Reply:
Yes... I will revert it back to what it was.

Reply:
this is a funky way to do stuff. types and refs are lost this way I IDEs which can cause untraceable errors. there should be no dynamic exports

Reply:
smth like this would do much better

```typescript
import * as v1beta3 from "./v1beta3";
import { mainnetId, testnetId, sandboxId } from "../constants";
import { getSelectedNetwork } from "@src/hooks/useSelectedNetwork";
import { NetworkId } from "@akashnetwork/akashjs/build/types/network";
export * from "./helpers";

const NETWORK_SDL: Record<NetworkId, typeof v1beta3> = {
  [mainnetId]: v1beta3,
  [testnetId]: v1beta3,
  [sandboxId]: v1beta3
};

export let selectedNetwork = getSelectedNetwork();
export let selectedNetworkId = selectedNetwork.id;
export const deploymentData = NETWORK_SDL[selectedNetworkId];
```

Reply:
Yea that works and looks better 👍